### PR TITLE
Update GenerateTests to use Fabulous.AST pre7

### PR DESCRIPTION
**Pull request by edgarfgp**
### WHAT
copilot:summary

copilot:poem

copilot:emoji

### HOW
copilot:walkthrough

### CODE REVIEW

**Comment by edgarfgp** [comment]
@TheAngryByrd Let me know if there is anything missing in Fabulous.AST :)

File: tests/GenerateTests.fs [hunk 1]
```diff
@@ -605,71 +571,68 @@ module GenerateTests =
 
     try
 
-      Record structure.Name { yield! expandFields structure }
+      Record(structure.Name) {
+        // expandFields structure
+        Field("TODO", LongIdent "TODO")
```
Comment (TheAngryByrd) [inline tests/GenerateTests.fs]: This was missing. I was able to fix it with this:

```fsharp
      Record(structure.Name) {
        yield! 
          expandFields structure
          |> List.map (fun (name, t) -> Field(name, t))
      }
```

**Review by TheAngryByrd — COMMENTED** [review]
Thanks for this!

File: tests/GenerateTests.fs (hunk 1 above)
Comment (edgarfgp) [reply tests/GenerateTests.fs]: Nice 👍

**Comment by TheAngryByrd** [comment]
All these changes are lovely btw :)

**Pull request merged**